vite-hardhat/package.json

```diff
@@ -23,8 +24,7 @@
     "@typechain/ethers-v5": "^10.1.0",
```

Comment:
wdyt about moving away from typechain and ethers to viem?

The Typechain README has [this](https://github.com/dethcrypto/TypeChain#soft-deprecation-notice) soft deprecation notice.

Reply:
done :)